spatialdb/spatialdb.py

```diff
@@ -552,7 +554,7 @@ def cutout(self, resource, corner, extent, resolution, time_sample_range=None, f
                         self.kvio.put_cubes(itemgetter(*s3_key_idx)(all_keys), temp_cubes)
 
             if len(zero_key_idx) > 0:
-                if not no_cache:
+                if  access_mode == "no_cache" or access_mode == "raw":
```

Comment:
Looking at the original if statement, this should be `if access_mode == "cache":`

Reply:
Good catch